cmd/run.go

```diff
@@ -440,4 +551,313 @@ func executeRunWithAppsConfigFile(runFilePath string) {
 		print.FailureStatusEvent(os.Stdout, "No apps to run")
 		os.Exit(1)
 	}
+	exitWithError, closeErr := executeRun(runFilePath, apps)
+	if exitWithError {
+		if closeErr != nil {
+			print.FailureStatusEvent(os.Stdout, fmt.Sprintf("Error closing resources: %s", closeErr))
```

Comment:
Do you think here using a `+` to concat string be useful? As performance of `+` seems better (one quick ref: https://dev.to/pmalhaire/concatenate-strings-in-golang-a-quick-benchmark-4ahh)

Reply:
will change this.

Reply:
done